CHANGELOG.md

```diff
@@ -11,6 +11,114 @@ git log --first-parent
 
 Please visit our wiki [Changelog](https://github.com/ginkgo-project/ginkgo/wiki/Changelog) for unreleased changes.
 
+## Version 1.9.0
+
+The Ginkgo team is proud to announce the new Ginkgo minor release 1.9.0.
+This release brings new features such as:
+- Supporting half precision (IEEE FP16). The type `gko::half` can now be selected in most instances as the value type
+  of a matrix, solver, preconditioner, etc. If the selected backend supports FP16 as a native type, the native type is
+  used within the kernels, otherwise an overhead might occur. The new behavior is enabled by default, but it can be
+  turned off during configuration.
+- Providing new implementations of the ILU and IC factorization for CUDA, HIP, OpenMP, and Reference backends. These are
+  available in addition to the existing implementations based on the vendor libraries cuSPARSE and hipSPARSE.
+- Adding new (S)SOR and Gauß-Seidel preconditioners.
```

Comment:
Probably better to use the standard spelling ? I guess Gauß is more accurate, but non-standard. In the PR list below it does say Gauss.
```suggestion
- Adding new (S)SOR and Gauss-Seidel preconditioners.
```